babel.config.js

```diff
@@ -3,30 +3,28 @@
 const envConfig = {
   ignoreBrowserslistConfig: true,
   modules: 'commonjs',
-  targets: { node: true }
```

Comment:
sorry, i accidentally prettified the babel config 😄

Reply:
haha no problem! thats a good thing